amass/sources/threatcrowd_test.go

```diff
@@ -0,0 +1,51 @@
+// +build datasources
+
+package sources
+
+import (
+	"log"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/OWASP/Amass/amass/core"
+)
+
+func TestThreaTCrowd(t *testing.T) {
```

Comment:
Typo: Function name should TestThreatCrowd

Reply:
Thanks